### PR TITLE
Swap RoomList to react-beautiful-dnd

### PR DESCRIPTION
**Pull request by lukebarnard1**

### CODE REVIEW

File: src/components/views/rooms/RoomList.js [hunk 1]
```diff
@@ -649,6 +747,7 @@ module.exports = React.createClass({
 
         const self = this;
         return (
+            <DragDropContext onDragEnd={this.onRoomTileEndDrag}>
```
Comment (ara4n) [inline src/components/views/rooms/RoomList.js]: Might be nice to fix the indentation here.

**Comment by ara4n** [comment]
looks plausible to me (modulo test fails) - is this going to break fa14bc9b8d2252f9d891d4eb54168fb0a272cd1c though?

**Comment by lukebarnard1** [comment]
> is this going to break fa14bc9 though?

I am not sure what that is or how it's related.

**Comment by ara4n** [comment]
lgtm

**Pull request merged**